tripleo-scripts/docker/trilio-datamover/trilio.filters

```diff
@@ -2,4 +2,7 @@
 # mount and unmout filter
 mount: CommandFilter, mount, root
 umount: CommandFilter, umount, root
+rescan-scsi-bus.sh: CommandFilter, /usr/bin/rescan-scsi-bus.sh, root
```

Comment:
We do not use these scripts for tripleo. tripleo dockerfiles are available in rhosp docker directory.